### PR TITLE
Increase activations test coverage to 100%

### PR DESCRIPTION
**Pull request by tleeuwenburg**
Increases the test coverage of activations.py to 100%

### CODE REVIEW

**Comment by tleeuwenburg** [comment]
Question ... do other developers wire up their forks directly to Travis for testing? My process has been to test on my machine for one Python version rather than hitting every single configuration/platform, and then using the merge request to put the change through its paces.

File: tests/keras/activations_test.py [hunk 1]
```diff
@@ -25,7 +27,32 @@ def test_serialization():
         assert fn == ref_fn
 
 
-def test_softmax():
+def test_get_fn():
+    '''
+    Activations has a convenience "get" function. All paths of this
+    function are tested here, although the behaviour in some instances
+    seems potentially surprising (e.g. situation 3)
```
Comment (fchollet) [inline tests/keras/activations_test.py]: Please format the docstring following conventions used elsewhere in the codebase.

File: tests/keras/activations_test.py [hunk 2]
```diff
@@ -42,6 +69,18 @@ def softmax(values):
     assert_allclose(result, expected, rtol=1e-05)
 
 
+def test_softmax_invalid():
+    '''
+    Test for the expected exception behaviour on invalid input
```
Comment (fchollet) [inline tests/keras/activations_test.py]: Same: docstring format

Comment (tleeuwenburg) [inline tests/keras/activations_test.py]: Sorry for these tidiness problems, I will try to reduce the effort for you to review in future.

**Pull request merged**